extensions/agroal/runtime/pom.xml

```diff
@@ -11,7 +11,7 @@
 
     <artifactId>quarkus-agroal</artifactId>
     <name>Quarkus - Agroal - Runtime</name>
-    <description>Pool JDBC database connections (included in Hibernate ORM)</description>
+    <description>JDBC Datasources and connection pooling</description>
```

Comment:
So FWIW, we were trying to unify descriptions with a verb.

Reply:
Thing is, it's not just about pooling, and people have been confused about this as well ("I don't need pooling, I use pgbouncer, let me get rid of that thing already!").

I can start with "Supports" of "Offers" if you want. Seems generic and verbose to me but I suppose it would be more consistent?